app/graph/types/project_media_type.rb

```diff
@@ -152,6 +152,14 @@
     }
   end
 
+  field :translations_count do
+    type types.Int
+
+    resolve ->(project_media, _args, _ctx) {
+      project_media.get_annotations('translation').count
+    }
+  end
```

Comment:
Yes... when `annotation_type` is present, return `get_annotations(annotation_type).count`, otherwise keep the current behavior.